iosHackthonApp/iosHackthonApp/Login/LoginView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  LoginView.swift
+//  iosHackthonApp
+//
+//  Created by Raynelle Francisca on 16/11/2020.
+//  Copyright © 2020 orgName. All rights reserved.
+//
+
+import SwiftUI
+import Strings
+import Theming
+
+struct LoginView: View {
+	@State var email: String = ""
+	@State var password: String = ""
+	@State var loginSucessful: Bool = true
+	let coloredNavAppearance = UINavigationBarAppearance()
+
+	init() {
+		coloredNavAppearance.configureWithOpaqueBackground()
+		coloredNavAppearance.backgroundColor = UIColor(ColorManager.appPrimary)
+		coloredNavAppearance.titleTextAttributes = [.foregroundColor: UIColor.white]
+		coloredNavAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor.white]
+		UINavigationBar.appearance().standardAppearance = coloredNavAppearance
+		UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
+	}
+
+	var body: some View {
+		NavigationView {
+			ScrollView {
+				VStack {
+					Image("") // Add proper logo
```

Comment:
Pretty picky I know, but can we add TODO to the beginning of your comment to make it really easy to search for them in the project please

Reply:
No worries..added a TODO